tutum.yml

```diff
@@ -1,11 +1,11 @@
 master:
-  image: 'citusdata/citus:5.2.2'
+  image: 'citusdata/citus:6.0.1'
```

Comment:
Leave this at `5.2.2` for now; this file is for Docker Cloud, which we haven't yet updated to understand the new Citus `pg_dist_node` stuff.